scripts/upgrade/nzbhydra.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+if [[ -f /tmp/.install.lock ]]; then
+  log="/root/logs/install.log"
+else
+  log="/root/logs/swizzin.log"
+fi
+
+. /etc/swizzin/sources/functions/utils
+active=$(systemctl is-active nzbhydra)
+username=$(_get_master_username)
+
+if [[ -d /opt/.venv/nzbhydra ]]; then
+    echo "NZBHydra v1 detected. Do you want to migrate data?"
+    echo 
+    echo "If you select no, a migration will not be attempted but your old data will be left."
+    select yn in "Yes" "No"; do
+        case $yn in
+            Yes ) migrate=True; majorupgrade=True; break;;
+            No ) migrate=False; majorupgrade=True; break;;
+        esac
+    done
+    if [[ $migrate == True ]]; then
+        echo "Do you wish to migrate your old database? If you select no, only settings will be transferred."
+        select yn in "Yes" "No"; do
+            case $yn in
+                Yes ) database=true; break;;
+                No ) database=false; break;;
+            esac
+        done
+        oldport=$(grep \"port\" /home/${username}/.config/nzbhydra/settings.cfg | grep -oP '\d+')
+        oldbase=$(grep \"urlBase\" /home/${username}/.config/nzbhydra/settings.cfg | cut -d\" -f4)
+        oldbaseconv=$(echo $oldbase | sed 's|/|%2F|g')
```

Comment:
Codacy found an issue: [See if you can use ${variable//search/replace} instead.](https://app.codacy.com/manual/flying_sausages/swizzin/pullRequest?prid=6214211)